doc/over_react_redux_documentation.md

```diff
@@ -657,4 +658,59 @@ var store = new DevToolsStore<AppState>(
   initialState: /*Default App State Object*/,
   middleware: [overReactReduxDevToolsMiddlewareFactory(name: 'Some Custom Instance Name')],
 )
-```
\ No newline at end of file
+```
+
+### Integration With DevTools
+
+In order to display the properties of Dart based `Action`s and `State` in the DevTools they must implement a `toJson` method. 
+
+`toJson` can be manually added to the classes, or added with the help of something like the the [json_serializable](https://pub.dev/packages/json_serializable) library. In the event that a value is not directly encodeable to `json`, we will make an attempt to call `toJson` on the value.
```

Comment:
Typo "the the"

Also, I think it'd be gook to mention  built_value's serializers, since that was the use-case that originally prompted this documentation ticket.

```suggestion
`toJson` can be manually added to the classes, or added with the help of packages like [json_serializable](https://pub.dev/packages/json_serializable) or [built_value](https://pub.dev/packages/built_value) and its serializers. In the event that a value is not directly encodeable to `json`, we will make an attempt to call `toJson` on the value.
```